docs/component-text.md

```diff
@@ -0,0 +1,32 @@
+# Component Text
+
+React Native Base Component Text
+
+## Text Props
+ this component extends from React Native Props.
+with some default Type Text
+
+## Usage
+
+```js
+import BaseText from 'rn-base-component';
```

Comment:
This import is not correct. Change to this instead
```suggestion
import {BaseText} from 'rn-base-component';
```

Reply:
updated